serializations.md

```diff
@@ -175,6 +170,8 @@ subsequent sections divided by object.
 | Channel | 58 |
 | Channel snapshot transaction | 59 |
 | POI | 60 |
+| Key block | 100 |
+| Micro block | 101 |
```

Comment:
Would you mind adding `| Light micro block | 102 |` or else we get a conflict there...

Reply:
I can do that.